content/paragraph/9954cc81-919b-4498-9151-bf930831fca7.json

```diff
@@ -83,7 +104,8 @@
         {
             "uri": "internal:\/",
             "title": "Button",
-            "options": []
+            "options": [],
+            "target_uuid": false
```

Comment:
I think that's correct. its not directly targeting an entity so the uuid is false. I think we want to keep it there since thats the accurate field structure.